tests/integration-tests/conftest.py

```diff
@@ -1081,11 +1086,64 @@ def vpc_stacks(cfn_stacks_factory, request):
             ],
         )
         template = NetworkTemplateBuilder(
-            vpc_configuration=vpc_config, default_availability_zone=availability_zones[0]
+            vpc_configuration=vpc_config,
+            default_availability_zone=availability_zones[0],
+            create_bastion_instance=True,
+            bastion_key_name=key_name,
+            region=region,
         ).build()
-        vpc_stacks[region] = _create_vpc_stack(request, template, region, cfn_stacks_factory)
+        vpc_stacks_dict[region] = _create_vpc_stack(request, template, region, cfn_stacks_factory)
+
+    return vpc_stacks_dict
+
+
+@pytest.fixture(scope="class")
+def vpc_stack_with_endpoints(region, cfn_stacks_factory, request, key_name):
+    """
+    Create a VPC stack with VPC endpoints.
+    Since VPC endpoints modify DNS at VPC level, all the subnets in that VPC will be affected.
+    :return: a VPC stack
+    """
+
+    credential = request.config.getoption("credential")
+    availability_zones = get_availability_zones(region, credential)  # FIXME
```

Comment:
-> Fixme: what there is to fix ?
Can we be explicit about the issue that we have to fix or remove the comment ?

Reply:
I'm going to improve the comment